velox/type/tests/HugeIntTest.cpp

```diff
@@ -97,3 +98,39 @@ TEST(HugeIntTest, parse) {
       testParse(hugeInt, "170141183460469231731687303715884105730"),
       "out of range of int128_t");
 }
+
+TEST(HugeIntTest, combine) {
+  auto testCombine =
+      [](int64_t first, int64_t second, const std::string& hugeString) {
+        EXPECT_EQ(std::to_string(HugeInt::combine(first, second)), hugeString);
+      };
+
+  testCombine(0, 0, "0");
+  testCombine(0, 13579, "13579");
+  testCombine(0, -13579, "-13579");
+  testCombine(1, 13579, "1" + std::string(13, '0') + "13579");
```

Comment:
Let's use "1000000000000013579" here and below for 13, 18 lengths. It is shorter.